docs/content/preview/explore/fault-tolerance/transaction-availability.md

```diff
@@ -0,0 +1,342 @@
+---
+title: Availability of Transactions
+headerTitle:  Availability of Transactions
+linkTitle: Availability of Transactions
+description: Simulate fault tolerance and resilience of transactions in a local YugabyteDB database universe.
+headcontent: Highly available and fault tolerant transactions
+menu:
+  preview:
+    identifier: transaction-availability-local
+    parent: fault-tolerance-1-macos
+    weight: 215
+type: docs
+---
+
+[Transactions](../../../architecture/transactions/distributed-txns/) are critical to many applications and need to work through different failure scenarios. YugabyteDB provides [high availability](../../../architecture/core-functions/high-availability/) (HA) of transactions by replicating the uncommitted values, a.k.a [provisional records](../../../architecture/transactions/distributed-txns/#provisional-records) across the [fault domains](../../../architecture/docdb-replication/replication/#fault-domains). The following examples demonstrate how YugabyteDB transactions survive common failure scenarios that could happen when a transaction is being processed. Some of the scenarios are:
```

Comment:
im still unclear on what we mean by "high availability of transactions". will defer reviewing the rest of this doc until we can sync on what this means

Reply:
Explained offline.